modules/apps/transfer/keeper/msg_server_test.go

```diff
@@ -94,15 +122,31 @@ func (suite *KeeperTestSuite) TestMsgTransfer() {
 
 			tc.malleate()
 
-			res, err := suite.chainA.GetSimApp().TransferKeeper.Transfer(sdk.WrapSDKContext(suite.chainA.GetContext()), msg)
+			ctx := suite.chainA.GetContext()
+			res, err := suite.chainA.GetSimApp().TransferKeeper.Transfer(sdk.WrapSDKContext(ctx), msg)
+
+			expEvents := map[string]map[string]string{
+				"ibc_transfer": {
```

Comment:
Instead of using the string values here, should we use instead the constants defined [here](https://github.com/cosmos/ibc-go/blob/main/modules/apps/transfer/types/events.go), so that if the string representation of the event or attribute changes, then we don't need to change the test.

Reply:
yes i was using it previously, but i wanted to hardcode the string itself, so that the tests breaks if the event type actually changes, since down stream applications could be utilising it.

Reply:
Cool, makes sense!